src/mainRecording.js

```diff
@@ -79,6 +79,7 @@ Vue.use(VueRouter)
 Vue.use(VueObserveVisibility)
 Vue.use(VueShortKey, { prevent: ['input', 'textarea', 'div'] })
 Vue.use(vOutsideEvents)
+Vue.directive('tooltip', Tooltip)
```

Comment:
I don't care for stable29, but in general - we should only use local registration, like with components. Otherwise it is hard to control whether a directive is available in a component or not.

Reply:
Thanks for future reference!
Yes, intent was only to fix in on stable29, since we drop it afterwards